libs/core/src/lib/select/select.component.scss

```diff
@@ -108,6 +108,10 @@
         pointer-events: none;
         margin-right: auto;
     }
+	
+    .icon-position{
+        margin: 1px 12px
+	}
```

Comment:
missing colon. We shoudl add more restrictive lint rules.

Reply:
fixed pls review @fkolar